### PR TITLE
React1 week1 files were added

### PR DESCRIPTION
**Pull request by Mabonye**
React1 week1 homework

### CODE REVIEW

File: react/react1/week1/vite-project/src/components/todoList/todoItem/todoItem.css [hunk 1]
```diff
@@ -0,0 +1,16 @@
+li {
```
Comment (LucasIversen) [inline react/react1/week1/vite-project/src/components/todoList/todoItem/todoItem.css:1]: Good to see you do some styling also 👍

File: react/react1/week1/vite-project/src/components/todoList/todoList.jsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
+/* eslint-disable react/prop-types */
+import React from "react";
+import TodoItem from "./todoItem/todoItem";
+
+function TodoList(props) {
+    return (
+        <ul>
+            {
+                props.todos.map((todo) => {
```
Comment (LucasIversen) [inline react/react1/week1/vite-project/src/components/todoList/todoList.jsx:9]: Good use of the map function

**Review by LucasIversen — APPROVED** [review]
Hi Jemima. Good job on your homework 💪 Keep it up

**Comment by Mabonye** [comment]
> Hi Jemima. Good job on your homework 💪 Keep it up

Hello Lucas, thanks for your time and comments:)

**Pull request merged**